BeNote/Home Tab/Register Screen/RegisterFirebaseManager.swift

```diff
@@ -61,8 +61,15 @@ extension RegisterViewController{
                 //MARK: the profile update is successful...
                 self.hideActivityIndicator()
                 
-                self.navigationController?.popViewController(animated: true)
-            }else{
+                DispatchQueue.main.async {
```

Comment:
same thing here for dispatch queue - if we want to make sure that the stuff inside runs in sequence after what's before it, I think it can just go outside after the self.hideActivity thing